src/main/features/claim/helpers/featuresBuilder.ts

```diff
@@ -37,6 +37,10 @@ export class FeaturesBuilder {
       if (FeatureToggles.isEnabled('directionsQuestionnaire') && await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_directions_questionnaire')) {
         features += features === '' ? 'directionsQuestionnaire' : ', directionsQuestionnaire'
       }
+
+      if (await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_judge_pilot')) {
```

Comment:
This is still in the wrong if. it should only be applied to cases above the LAPilotEligible threshold